tests/apollo/util/bft.py

```diff
@@ -407,10 +423,10 @@ def start_cre(self):
                     now = datetime.now().strftime("%y-%m-%d_%H:%M:%S")
                     test_name = f"{now}_{self.current_test}"
 
-                self.test_dir = f"{self.builddir}/tests/apollo/logs/{test_name}/{self.current_test}/"
-                test_log = f"{self.test_dir}stdout_cre.log"
+                self.current_test_case_path = f"{self.builddir}/tests/apollo/logs/{test_name}/{self.current_test}/"
```

Comment:
This requires a full transofmration of all paths in bft.py to pathlib since it is used in many places.
What I did here is only to give better names and refactor some code.